hazelcast-client/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -359,6 +363,21 @@ private void handleFlakeIdGenerator(Node node) {
         clientConfig.addFlakeIdGeneratorConfig(config);
     }
 
+    private void handleReliableTopic(Node node) {
```

Comment:
There are two more setters in ReliableTopicConfig class (messageListenerConfigs, statisticsEnabled). Can we put the others as well? Would it be possible?

Reply:
@ihsandemir I think you are looking at wrong class. ClientReliableTopicConfig is the one that you should consider, not ReliableTopicConfig. That is on the server side.